packages/cli/src/cmds/validator/voluntaryExit.ts

```diff
@@ -97,7 +97,11 @@ ${validatorsToExit.map((v) => `${v.pubkey} ${v.index} ${v.status}`).join("\n")}`
     }
 
     for (const [i, {index, signer, pubkey}] of validatorsToExit.entries()) {
-      const domain = config.getDomain(computeStartSlotAtEpoch(exitEpoch), DOMAIN_VOLUNTARY_EXIT);
+      // Deneb onwards the signature domain fork is fixed to Capella version
```

Comment:
there are duplicate logics in 3 places, should we use a new util `getDomainForVoluntaryExit()` somewhere @g11tech ?

Reply:
makes sense, will do the change